tests/test_videoframe.py

```diff
@@ -49,6 +49,24 @@ def test_opaque() -> None:
                 assert type(frame.opaque) is tuple and len(frame.opaque) == 2
 
 
+def test_frame_duration_matches_packet() -> None:
+    with av.open(fate_suite("h264/interlaced_crop.mp4")) as container:
```

Comment:
Question for maintainers: is there a more interesting video we could put here? This one seems to have `duration=1` for all packets

Reply:
Something with VFR would do. This test is fine though, considering how annoying, at times, finding stuff in FATE can be.

Reply:
This can be added in another PR

Reply:
OK thanks, will look into the FATE videos and find one that works for a followup PR